navigation/MainNavigationContext.ts

```diff
@@ -0,0 +1,18 @@
+import { createContext } from "react";
+import { Episode } from "../types";
+
+// All screens that exist that do not need props passed to them, if props need to be passed, add it to NavigationState
+export enum NavigationScreens {
+  intro,
+  createEpisode,
+  predictions,
+}
+
+// All possible states that we can be in. Add your navigation state here for new screens
```

Comment:
interpretations for what state {episodes: Episode[] } and {predictions: String[]} for clarity👀 The NavigationScreens enum is pretty clear since they're all string but might be nice there too